resources/dicts/patrols/general/border.json

```diff
@@ -182,6 +182,29 @@
     "history_text": [
         "This cat got a scar from a fight with a small dog, but they always say it was a huge dog."
     ]
+},
+{
+    "patrol_id": "gen_bord_abandonedkittypet1",
+    "biome": "Any",
+    "season": "Any",
+    "tags": ["border", "new_cat_injury_kittypet", "nc_sickness",  "respect", "warrior"],
+    "intro_text": "The patrol hears the wailing of a cat and the sound of a monster speeding away.",
+    "decline_text": "It's none of their business really and they have to finish the border patrol.",
+    "chance_of_success": 30,
+    "exp": 15,
+    "success_text": [
+    "They find a well groomed kittypet collapsed by the thunderpath wailing for their twolegs to come back. p_l comforts them and when asked why their twolegs would do this, the only explanation that they can give is that they got sick recently. p_l offers the former kittypet a place in c_n till they are better at least. They tentatively accept whilst looking longely back to the thunderpath."
```

Comment:
Some grammar and spelling issues, could fix to:
    "They find a well groomed kittypet collapsed by the thunderpath, wailing for their **Twolegs** to come back. p_l comforts them **and asks why the Twolegs have left them.** **The** only explanation that they can give is that **they've been sick recently, and perhaps a burden?** p_l offers the former kittypet a place in c_n, till **they're** better at least. They tentatively accept whilst looking **longingly** back to the thunderpath."